Codemine/Extensions/CGRect+Utilities.swift

```diff
@@ -35,7 +35,7 @@ public extension CGRect {
      
      - returns: a new CGRect with the width and height reversed to those of the current one
      */
-    public func rectByReversingSize() -> CGRect {
+	public var reversedSize: CGRect {
```

Comment:
What about reversingSize? That seems to be similar to the examples here https://github.com/apple/swift-evolution/blob/master/proposals/0005-objective-c-name-translation.md `func bezierPathByReversingPath() -> UIBezierPath` -> `func reversing() -> UIBezierPath`